### ISSUE TITLE
[FEATURE] New Language

### ISSUE THREAD

**Issue by Kauziin** [Osaxely/mc_addons_manager#7]
I'm a brazilian user and my with the app is a little difficult...
I would be wondering if i could help with the Portuguese Translation

**Comment by Osaxely** [comment]
Hi ! I saw your email and then this issue.
Yes, you are welcomed to contribute to add your language :)
I’m curently not at home, so I will give you simple steps to add your language to the repository (if you dont know how to)

**Comment by Kauziin** [comment]
Ok

**Comment by Osaxely** [comment]
@Kauziin So I'm here :) Sorry for the late response
I've seen that you have forked the project, it's a good start ! The next step is to add the translation file to your fork, to do so, go to Strings folder and create folder called pt-br if you're on brazil, or pt-pt if portugual, next be sure that the filename is Resources.resw
After that, push to your repository the new created folder with the file in, and go to [Pull Request](https://github.com/Kauziin/mc_addons_manager/pulls), then click on New pull request. On base repository set your fork, and on head repository select my repository.
Then it will create the request and all will be good :) If you have any questions don't hesitate to ask !

**Comment by Kauziin** [comment]
how do you create folders on that site?

Em seg, 23 de nov de 2020 16:55, Osaxely <notifications@github.com>
escreveu:

**Comment by Osaxely** [comment]
Download the app [Github Desktop](https://desktop.github.com/), when you launch the app, select File -> Clone repository... click on your repository and click on Clone button.
![image](https://user-images.githubusercontent.com/59126998/100012082-ffa02700-2dd2-11eb-840e-133b9658e002.png)

When the repo will be cloned, be sure that it is in Current repository
![image](https://user-images.githubusercontent.com/59126998/100012011-e39c8580-2dd2-11eb-8b69-6e5f0e1e5a4f.png)
Then select Repository -> Show in file explorer (or CTRL-SHIFT-F)
Normally you'll have something like that
![image](https://user-images.githubusercontent.com/59126998/100012185-25c5c700-2dd3-11eb-8557-da85fd9fce63.png)
On this, go to Strings folder, create the folder and put your Resources.resw in.
Then go to Github Desktop, it normally show that files were changed here :
![image](https://user-images.githubusercontent.com/59126998/100012259-468e1c80-2dd3-11eb-860a-3d8f9a241be9.png)
So in Summary enter the new thing (Added portuguese translation file for exemple), and commit to master.
Then you'll be good to follow the other steps that I mentionned in my last message (Pull request)
If you don't have a PC I will explain how to do directly on the github website (But it's less easier :D)

**Comment by Kauziin** [comment]
I think I managed to do it by phone a little faster :)

Em seg, 23 de nov de 2020 17:33, Osaxely <notifications@github.com>
escreveu:

**Comment by Osaxely** [comment]
Perfect, I see that you have commited the file and then deleted it (Github doesn't support empty folder so it was deleted)

**Comment by Kauziin** [comment]
I identified an error in the file so I had to delete and redo the file...

I will resend a file, wait a minute.

Em seg, 23 de nov de 2020 17:43, Kauziin Extra <kaue191006@gmail.com>
escreveu:

**Comment by Osaxely** [comment]
Ok ! Perfect :) What are you using to edit the file ?

**Comment by Kauziin** [comment]
an application that opens various types of texts and scripts, and is a file
explorer.

Em seg, 23 de nov de 2020 18:10, Osaxely <notifications@github.com>
escreveu:

**Comment by Osaxely** [comment]
Ok ! It's a good way to edit this files 👍 
Normally it can be edited very easily with Visual Studio (but not on phone :/), with just cells like this 
![image](https://user-images.githubusercontent.com/59126998/100016512-d33bd900-2dd9-11eb-8381-c37f567c16f4.png)
If app localization become more important I will try to create an app to edit this file more easily

**Comment by Osaxely** [comment]
Just to explain precisely, you have probably seen that every applications strings looks like that :
```
  <data name="StringKey" xml:space="preserve">
    <value>Translated string :</value>
    <comment>Comment, the app doesn't load it, it's just for the translators to know what they do</comment>
  </data>
```
And when the app want a localized text, it just call by the "ID" (The StringKey), and this file return the text for the correct language

**Comment by Osaxely** [comment]
The file is on repo, can you make a pull request please ? :)
To do that, click [here](https://github.com/Kauziin/mc_addons_manager/pulls), then select New pull request.
earlier I was wrong : The head repository must be your fork, and the base my repository !
It will be like this :
![image](https://user-images.githubusercontent.com/59126998/100017541-5c074480-2ddb-11eb-8c86-dba8324d51f3.png)

**Comment by Osaxely** [comment]
Perfect ! Thank you very much for your contribution :)
Portuguese will probably be available on the next update thanks to you !

**Comment by Kauziin** [comment]
You're welcome! '-'

Em seg, 23 de nov de 2020 18:48, Osaxely <notifications@github.com>
escreveu:
